### DISCUSSION TITLE
Are there any ETA when first standalone release come?

### DISCUSSION THREAD

**Discussion by ViBE-HU** [input-leap/input-leap#1475] [Q&A]
It is a pain to follow what's going on IRC (cuz usually nothing), so I had no better idea to ask my question.

Barrier is nearly dead and it's useless on several distros. Even the last snap build is cr#p, no mouse pointer on the client. Guess it's a Wayland related issue or who knows. So are there any chance that a working Input Leap(?) will be released soon? I'm really curious and would love to see a working solution.

**Comment by shymega** [comment]
Still lots to be done to migrate references to Barrier, I want to merge some PRs from Barrier's repo, but I'm busy with work ....

**Reply by ViBE-HU** [reply]
Sounds confusing but thanks the infos.

**Comment by mariomadproductions** [comment]
Some unofficial builds I've made, following these build instructions:
https://github.com/input-leap/input-leap/wiki/Building-on-Windows
https://github.com/input-leap/input-leap/wiki/Building-on-Linux

Commit: a9ffcbe

Linux Mint 20.3: 
[input-leap-a9ffcbe-linux-mint-20.3.7z.zip](https://github.com/input-leap/input-leap/files/9192427/input-leap-a9ffcbe-linux-mint-20.3.7z.zip)

Windows 8.1: 
[input-leap-a9ffcbe-windows-8.1.7z.zip](https://github.com/input-leap/input-leap/files/9192430/input-leap-a9ffcbe-windows-8.1.7z.zip)

virustotal:
https://www.virustotal.com/gui/file/6407763080eea83c6292b614b3f274ec3c17b99670642f61d43e094d7fd39ffb
https://www.virustotal.com/gui/file/1cdc827056b23616671ed17f956a89f5e3d2b759df8e178182cc63da6ed1d77f
https://www.virustotal.com/gui/file/59eaa7dfa56a77b13490c16d51caa5c08b00ca8745469c360bb481f55bd50999
https://www.virustotal.com/gui/file/59ffad3ed4f82a925006e1361a2378628413a2972dd9f6b5db4caf89ba7b3754

Not 100% sure if it works though. They start up but I can't get the windows and linux pc to connect.

**Reply by ViBE-HU** [reply]
Did a quick test... I did exactly on Windows 8.1 and it does not even start because a missing DLL. On Ubuntu it starts, but noted some glitches. No languages. Even crashed by selecting a language. The "taskbar" icon does not shows if the connection is active.

**Reply by ViBE-HU** [reply]
There is an old glitch btw. If the host is Linux and the client is Windows you cannot type "@" sign on the client. So the actual code is still affected by this bug.

**Reply by mariomadproductions** [reply]
Strange.

**Reply by hadallen** [reply]
I had to use `sudo cmake -P cmake_install.cmake` to install binaries built from source on debian

**Reply by hadallen** [reply]
and on windows11 I had to build installers to install the barrier service properly

**Reply by mariomadproductions** [reply]
Both my systems had synergy and barrier installed on them previously (and maybe at the same time), so that might have been why it worked.

**Reply by hadallen** [reply]
yes that makes sense, it was working on my win11 until I uninstalled the barrier release

**Reply by KaKi87** [reply]
Hello,
Could you please make builds for Linux Mint 19.3 and Windows 10 ?
Thanks

**Reply by mariomadproductions** [reply]
> Hello, Could you please make builds for Linux Mint 19.3 and Windows 10 ? Thanks

Sorry, I don't have time to do that, but if you get stuck following the instructions I followed, I'm happy to give some pointers.
